spec/Store.spec.js

```diff
@@ -545,6 +574,282 @@ describe('Store', () => {
     })
   })
 
+  describe('findMany', () => {
+    describe('"fromServer" is set to false', () => {
+      describe('records of the specified type do not exist', () => {
+        it('returns an empty array', () => {
+          const todos = store.findMany('todos', ['1001', '5000'], { fromServer: false })
+          expect(todos).toHaveLength(0)
+        })
+      })
+
+      describe('records of the specified type do exist', () => {
+        it('returns existing only models in the store', () => {
+          store.add('todos', createMockTodosAttributes(5, '1000'))
+          const todos = store.findMany('todos', ['1001', '5000'], {
+            fromServer: false
+          })
+          expect(todos).toHaveLength(1)
+        })
+      })
+    })
+
+    describe('when "fromServer" is set to true', () => {
+      it('fetches data from server', async () => {
+        expect.assertions(4)
+        fetch.mockResponse(createMockTodosResponse(5, '1000'))
+        const ids = createMockIds(5, '1000')
+        const todos = await store.findMany('todos', ids, { fromServer: true })
+        expect(todos).toHaveLength(5)
+        expect(todos[0].title).toEqual('Todo 1000')
+        expect(fetch.mock.calls).toHaveLength(1)
+        expect(fetch.mock.calls[0][0])
+          .toEqual('/example_api/todos?filter%5Bids%5D=1000%2C1001%2C1002%2C1003%2C1004')
+      })
+
+      it('uses multiple fetches for data from server', async () => {
+        expect.assertions(7)
+
+        fetch.mockResponseOnce(createMockTodosResponse(100, '1000'))
+        fetch.mockResponseOnce(createMockTodosResponse(100, '1100'))
+        fetch.mockResponseOnce(createMockTodosResponse(100, '1200'))
+
+        const ids = createMockIds(300, '1000')
+        const todos = await store.findMany('todos', ids, { fromServer: true })
+
+        expect(todos).toHaveLength(300)
+        expect(store.findAll('todos', { fromServer: false })).toHaveLength(300)
+
+        expect(fetch.mock.calls).toHaveLength(3)
+        const [firstCall] = fetch.mock.calls[0]
+        expect(decodeURIComponent(firstCall)).toMatch(/1139$/)
```

Comment:
what is the significance of this match?

Reply:
the url param should be split so that 1139 is the last one in the list (1140 would be the first one in the next list)